kubernetes/gitlab-auth.sh

```diff
@@ -49,14 +49,13 @@ getGitlabToken() {
     | sed 's/.*<form class="new_user[^<]*\(<[^<]*\)\{2\}authenticity_token" value="\([^ ]*\)".*/\2/' \
     | sed -n 1p)
 
-  curl -b "$COOKIES_FILE" -c "$COOKIES_FILE" -s --output /dev/null \
+  curl -b "$COOKIES_FILE" -c "$COOKIES_FILE" -s --output=/dev/null \
     -i "$GITLAB_URL/users/sign_in" \
-    --data "user[login]=$GITLAB_USER&user[password]=$GITLAB_PASS" \
-    --data-urlencode "authenticity_token=$csrfToken"
+    --data="user[login]=$GITLAB_USER&user[password]=$GITLAB_PASS" \
```

Comment:
@tyranron, it won't work. [curl docs](https://curl.haxx.se/docs/manpage.html#-d)